bool mapping_request
---
string[] occFields
